x/upgrade/module.go

```diff
@@ -0,0 +1,129 @@
+package upgrade
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/celestiaorg/celestia-app/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/upgrade/client/cli"
+)
+
+func init() {
+	types.RegisterLegacyAminoCodec(codec.NewLegacyAmino())
+}
+
+const (
+	consensusVersion uint64 = 2
```

Comment:
This was just directly taken from the existing upgrade module (since it shares the same name). It should probably actually be 3 instead of 2 now that I think of it